libraries/postgres/src/Schema.js

```diff
@@ -12,7 +13,7 @@ class Schema{
    * the Postgres DO statment; that is usually 'BEGIN stmt; stmt; .. END'.
    */
   // TODO: Make sure that procedure argument values don't change
-  constructor(versions, access) {
+  constructor(versions = schemaJson.versions, access = schemaJson.access) {
```

Comment:
I have a vague preference that reference to the generated bits occurs in the `db/` directory, and that this library remains agnostic to it.  So services would call a method in `taskcluster-db` to construct a new Database, and that would automatically load a schema from the generated schema.